tools/ka-customizer

```diff
@@ -181,12 +234,14 @@ done_button.connect('clicked', () => {
         args.push(' --dummy-content');
     win.get_window().cursor = Gdk.Cursor.new_for_display(Gdk.Display.get_default(), Gdk.CursorType.WATCH);
     win.set_sensitive(false);
-    let [, pid] = GLib.spawn_async(null, args, null, GLib.SpawnFlags.DO_NOT_REAP_CHILD, null, null);
-    GLib.child_watch_add(GLib.PRIORITY_HIGH, pid, (pid, status) => {
+    let app_process = Gio.Subprocess.new(args, Gio.SubprocessFlags.STDOUT_PIPE | Gio.SubprocessFlags.STDERR_MERGE);
+    app_process.wait_async(null, (app_process, result)  => {
+        app_process.wait_finish(result);
         win.get_window().cursor = null;
         win.set_sensitive(true);
-        if (!GLib.spawn_check_exit_status(status))
-            show_message('Something went wrong :(. Ask a developer.');
+        if (app_process.get_if_exited() && app_process.get_exit_status() === 0)
```

Comment:
Because ka-customizer runs everything against the actual flatpak runtimes, and the runtimes won't be updated to exit(1) on on a parse errors until this is merged, its probably easiest to comment out this check to see the error message dialog.